customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
+        }
+      }).bind(this)
+    )
+    .then(
       function () {},
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
```

Comment:
> into a shared password-checking helper function somewhere.

I was thinking about this too. The functionality doesn't seem to make sense in crypto/password.js because then we'd introduce extra deps there. A helper file in the root or `/routes`?